vyper/ast/grammar.lark

```diff
@@ -101,6 +102,8 @@ _INTERFACE_DECL: "interface"
 mutability: NAME
 interface_function: function_sig ":" mutability
 interface_def: _INTERFACE_DECL NAME ":" _NEWLINE _INDENT ( interface_function _NEWLINE)+ _DEDENT
+_IMPLEMENTS_DECL: "implements"
+implements_def: _IMPLEMENTS_DECL ":" ( _import_name | _import_alias_name )
```

Comment:
why not just `_IMPLEMENTS_DECL ":" NAME`?

Reply:
I thought the RHS of `_IMPLEMENTS_DECL ":" NAME` should be limited to the identifier(s) that have been imported into the namespace, which would be (1) `_import_name`; and (2) the `NAME` in `import_alias: _AS NAME`. 

I think `_IMPLEMENTS_DECL ":" NAME` is more general, but it could also work since the typechecker would catch an identifier that has not been imported.

Reply:
well, the LHS is effectively just an alias for the RHS, for readability. it is not like the parser is going to check that the identifiers have been included in import statements, that happens during semantic analysis anyways.

Reply:
oh okay got it, will make the change!